pkg/ruler/base/api.go

```diff
@@ -448,14 +448,16 @@ func (a *API) CreateRuleGroup(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	logger = log.With(logger, "namespace", namespace, "tenant", userID, "userID", userID)
```

Comment:
Is it really necessary to add the new `tenant` label? I assume it is added to be more consistent across logs, and the old `userID` is kept to be backwards compatible?

Reply:
I believe we normally log the tenant ID under the `user` field

Reply:
Unfortunately, we are pretty inconsistent with that. We have `user`, `org_id`, `userID`, `tenant`.

Reply:
I _think_ we use `user` the most, but yes we need to make this consistent. We have a task for including the tenant ID in all log lines, and in that task we can make it consistent.

In any case, I don't see the need for both `tenant` and `userID` here.

Reply:
> Unfortunately, we are pretty inconsistent with that. We have user, org_id, userID, tenant.

That was my thought. I'll remove `tenant`